src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -396,7 +381,8 @@ export const useBalanceChartData: UseBalanceChartData = args => {
     // data prep
     const noDeviceId = isNil(walletInfo?.deviceId)
     const noAssetIds = !assetIds.length
-    if (noDeviceId || noAssetIds || priceHistoryDataLoading) {
+    const noPriceHistoryData = isEmpty(priceHistoryData)
+    if (noDeviceId || noAssetIds || priceHistoryDataLoading || noPriceHistoryData) {
```

Comment:
this was obscure and previously wrong, we'd try and compute without price history data, type def update caught this